### PR TITLE
Obsolete information in README file is updated

### PR DESCRIPTION
**Pull request by Chillli**
Obsolete information for Autopsy 3 is removed. And it is rewrited in markdown with current information for Autopsy 4.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
+March 3, 2016
 
 
-OVERVIEW
+
+##OVERVIEW
 
 Autopsy is a graphical interface to The Sleuth Kit and other open source digital forensics tools. 
-Autopsy 3 is a complete rewrite from Autopsy 2 and it is now Java-based.     
+Autopsy 4 (and 3) are a complete rewrite from Autopsy 2, and none of this document is relevant to Autopsy 2.
```
Comment (rcordovano) [inline README.md]: I would prefer to say:

Autopsy 4 improves on Autopsy 3 by supporting collaboration on a single case by multiple users. Autopsy 3 was a complete rewrite of Autopsy 2 using Java, and the information presented in this document is not relevant to Autopsy 2.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,32 @@
...
+
+##OVERVIEW
 
 Autopsy is a graphical interface to The Sleuth Kit and other open source digital forensics tools. 
-Autopsy 3 is a complete rewrite from Autopsy 2 and it is now Java-based.     
+Autopsy 4 (and 3) are a complete rewrite from Autopsy 2, and none of this document is relevant to Autopsy 2.   
 
 Although Autopsy is designed to be cross-platform (Windows, Linux, MacOSX), the current version is fully functional and fully tested only on Windows. 
-We have run it on XP, Vista, and Windows 7 with no problems. 
+You can run Autopsy 4 on Linux and OS X, but it must be built from source code.
```
Comment (rcordovano) [inline README.md]: Why delete the statement that we have run Autopsy on Windows XP, Vista, and Windows 7? Note that as far as I know, no one here at Basis Technology has tried to run Autopsy on Windows 8 or 10.

Autopsy is intended to be cross-platform, but have you built Autopsy for Linux and/or OS-X and run it successfully? I have heard reports of Linux builds and we are planning to attempt an OS-X build in the near future, so I would be very interested in hearing about your experiences if you have done these builds. If not, we should not make any stronger claims about Linux and OS-X builds until it is verified that they can be done, at which time we may wish to augment this document with Linux and OS-X build details.

Comment (millmanorama) [inline README.md]: I have run Autopsy on Windows 8 (at least through the IDE)

File: README.md [hunk 3]
```diff
@@ -1,24 +1,32 @@
...
-Autopsy 3.0 is released under the Apache 2.0 license. 
+Autopsy 3 and 4 source code are distributed under a Apache 2 license. The package contains libraries that may have different licenses.
 
 
-INSTALLATION
+##INSTALLATION
 
 All Autopsy dependencies are bundled with the installer provided.
-There is no need for manual installation of additional dependencies if the installer is used.
+For Autopsy 3, there is no need for manual installation of additional dependencies if the installer is used.
```
Comment (rcordovano) [inline README.md]: I don't think we need to make a distinction between Autopsy 3 and 4 here.

File: README.md [hunk 4]
```diff
@@ -1,24 +1,32 @@
...
+The current version of Autopsy 4 is distributed on sleuthkit.org only as a Windows installer. It can run on Linux and OS X, but requires some manual setup. To install Autopsy, perform the following steps:
+
```
Comment (rcordovano) [inline README.md]: Please refer to prior comments - I'm not sure what the manual setup might be. Is this also said somewhere on th web site?

File: README.md [hunk 5]
```diff
@@ -1,24 +1,32 @@
...
+* Run the Autopsy msi file
+* If Windows prompts with User Account Control, click Yes
+* Click through the dialog boxes until you click a button that says Finish
+* Autopsy should now be fully installed
+
```
Comment (rcordovano) [inline README.md]: I think we have historically assumed the user knows how to run a Windows installer, so we really don't need to include these lines.

**Comment by ruchambapat** [comment]
Thank you for your comments!! We agree with your suggestions.

**Pull request merged**